### ISSUE TITLE
How to reformat the input paras in pretrained FCN .caffemodel?

### ISSUE THREAD

**Issue by m5061125** [NVIDIA/DIGITS#1491]
Hello, I am try to reuse semantic-segmentation exple with my own data, which is 512by512 dicom file, after build the database (convert to 3-channel) and start the train with pretrained alexnet .caffemodel, the system tell me :+1: 
Creating layer score_score_0_split
Creating Layer score_score_0_split
score_score_0_split <- score
score_score_0_split -> score_score_0_split_0
score_score_0_split -> score_score_0_split_1
Setting up score_score_0_split
Top shape: 1 2 512 512 (524288)
Top shape: 1 2 512 512 (524288)
Memory required for data: 118994192
Creating layer loss
Creating Layer loss
loss <- score_score_0_split_0
loss <- label_label_0_split_0
loss -> loss
Creating layer loss
Setting up loss
Top shape: (1)
with loss weight 1
Memory required for data: 118994196
Creating layer dice

I use ubuntu 14.04, digits 5.1-dev, nvcaffe 0.15.14, GPU is 1070, 8G memory, and 32G RAM for CPU.
I am wondering, if I can reformat the .coffemodel into one channel (instead of 3), then I can proceed the training. So does anyone know the tricks, thanks a lot.

**Comment by m5061125** [comment]
It is interested, when I remove the dice layer, the training can proceed,
Hardware
GeForce GTX 1070 (#0)
Memory
    1.47 GB / 7.92 GB (18.5%) 
GPU Utilization
    100%
Temperature
    71 °C
Process #6449
CPU Utilization
    113.9%
Memory
    11.4 GB (36.4%) 
Anyone know the reasons?

**Comment by adnphllps** [comment]
Does your training data have more than 2 classes? Dice will only work for binary segmentation.

**Comment by m5061125** [comment]
Thansk adnphillps. Just two class, one is organ, one is background. I will look throught the code to see what I can do, thanks again.

**Comment by m5061125** [comment]
I solve it using net.surgery, thus close this item.
